database/table_creation/table_teams.sql

```diff
@@ -33,7 +36,6 @@ SELECT
     twitter,
     (SELECT city_id FROM cities WHERE name = t.city) AS city_id,
     (SELECT arena_id FROM arenas WHERE name = t.arena) AS arena_id
```

Comment:
I guess both, city's name and new added name confilicted there, should be defined more clearly.